### ISSUE TITLE
Game Support - Knights of the Old Republic I & II

### ISSUE THREAD

**Issue by erri120** [Nexus-Mods/NexusMods.App#367]

**Comment by th3w1zard1** [comment]
KOTOR-specific information on this issue was discussed in the r/KOTOR discord. A copy of the discussion itself can be found here:
https://pastebin.com/z1SqxGzZ

**Comment by th3w1zard1** [comment]
I ran into significant inconsistencies with the default INI syntax and the way TSLPatcher (the main installer for KOTOR mods) parses its `changes.ini` file. Perhaps referencing these inconsistencies would help in some way? Info can be found here:

https://github.com/NickHugi/PyKotor/pull/7

This is a PR I submitted to another project capable of parsing the `changes.ini` file TSLPatcher uses to define instructions. The jist of the problem is a lot of the ini files don't need to have certain fields or even header information defined to be a valid TSLPatcher changes.ini file.

Basically, tslpatcher is extremely lenient on syntax which makes creating 3rd party libraries capable of parsing it a bit problematic, as it's difficult to determine what exactly might fail to parse unless you test out hundreds of mods. Luckily, my project http://github.com/th3w1zard1/KOTORModSync is designed to install hundreds of mods, so I was able to iron a lot of these problems out in the PyKotor PR above.

**Comment by erri120** [comment]
> https://github.com/NickHugi/KotorDotNET

Thanks, will take a look.
